oshdb/src/main/java/org/heigit/bigspatialdata/oshdb/osm/OSMEntity.java

```diff
@@ -52,8 +52,16 @@ public OSHDBTimestamp getTimestamp() {
     return timestamp;
   }
 
+  public long getChangesetId() {
+	    return changesetId;
+	  }
```

Comment:
looks like there are some superfluous tabulators above

```suggestion
    return changesetId;
  }
```